#! /bin/sh /usr/share/dpatch/dpatch-run
## log-static-142322.dpatch by Marc Haber <mh+debian-packages@zugschlus.de>
##
## All lines beginning with `## DP:' are a description of the patch.
## DP: Function log should be static, see #142322.

@DPATCH@
diff -urNad --exclude=CVS --exclude=.svn ./snoopy.c /tmp/dpep-work.anNBjg/snoopy/snoopy.c
--- ./snoopy.c	2000-12-21 07:03:05.000000000 +0000
+++ /tmp/dpep-work.anNBjg/snoopy/snoopy.c	2005-06-18 11:17:11.585065776 +0000
@@ -37,7 +37,7 @@
 
 #define FN(ptr,type,name,args)  ptr = (type (*)args)dlsym (REAL_LIBC, name)
 
-inline void log(const char *filename, char **argv) {
+static inline void log(const char *filename, char **argv) {
 	static char *logstring=NULL; 
 	static int argc, size=0;
 	register int i, spos=0;
